### PR TITLE
INTMDB-535-adding execution role and documentation update

### PR DESCRIPTION
**Pull request by govardhanpagidi**
## Description

adding execution role and documentation update

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: cfn-resources/README.md [hunk 1]
```diff
@@ -7,21 +7,42 @@ MongoDB Atlas Resource Types.
 
 ### Resource Status Table
 
-| Resource                             | Status                                                 | Examples                                                                                   |
```
Comment (Zuhairahmed) [inline cfn-resources/README.md:10]: can you add 3 BLOCKED resources as well to this table (use the Status unstable) and link to source code?

**Review by Zuhairahmed — CHANGES_REQUESTED** [review]
Thanks @govardhanpagidi. Few comments/commits below. Also can you double check  examples/execution-role.yaml since this folder is empty.

File: cfn-resources/README.md [hunk 2]
```diff
@@ -1,27 +1,50 @@
...
-MongoDB Atlas Resource Types.
+MongoDB Atlas Resources. 
+
+Note these are also hosted on AWS CloudFormation Public Registry under Third Party Extensions. 
 
 ### Resource Status Table
 
-| Resource                             | Status                                                 | Examples                                                                                   |
-|--------------------------------------|--------------------------------------------------------|--------------------------------------------------------------------------------------------|
-| project                              | ![Build](https://img.shields.io/badge/Beta-yellow)     | [./project/test](./project/test)                                                           |
```
Comment (Zuhairahmed) [inline cfn-resources/README.md:12]: can you confirm that there were indeed 33 resources developed (including Cloud Provider Access)? google sheets tracker appears to have a few extra

**Review by ParthasarathyVaradhan — COMMENTED** [review]
examples/execution-role.yaml file seems to me empty, is it meant to be empty ?

File: cfn-resources/README.md (hunk 2 above)
Comment (govardhanpagidi) [reply cfn-resources/README.md:12]: There are total of 37 resources (excluded the redos). Out of them 33 are published, 1 is not published and 3 are not implemented lack of APIs.

**Comment by govardhanpagidi** [comment]
@ParthasarathyVaradhan  Execution Role content updated

File: cfn-resources/README.md (hunk 1 above)
Comment (govardhanpagidi) [reply cfn-resources/README.md:10]: Updated

File: examples/execution-role.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This CloudFormation template creates a role assumed by CloudFormation
```
Comment (Zuhairahmed) [inline examples/execution-role.yaml:3]: just to confirm this 1 execution role can be used to deploy all 33 Atlas resources that have been published to CFN Public Registry?

File: cfn-resources/README.md (hunk 1 above)
Comment (Zuhairahmed) [reply cfn-resources/README.md:10]: I don't see them? also looks like this table reverted back to older version (i made a few commits earlier today to clean up)

**Review by ParthasarathyVaradhan — COMMENTED** [review]
cfn-resources/cluster/rpdk.log.1 this file is not needed for this commit

File: examples/execution-role.yaml (hunk 3 above)
Comment (govardhanpagidi) [reply examples/execution-role.yaml:3]: Yes, this Role can be used for all resources.

**Comment by govardhanpagidi** [comment]
> cfn-resources/cluster/rpdk.log.1 this file is not needed for this commit

Removed

**Review by ParthasarathyVaradhan — APPROVED** [review]
LGTM

**Review by Zuhairahmed — APPROVED** [review]
LGTM! Thank you @govardhanpagidi!!

**Pull request merged**